src/foam/nanos/cron/Schedulable.js

```diff
@@ -347,16 +347,17 @@ foam.CLASS({
         var today = new Date();
 
         if ( lastRun == null ) return true;
+
         var alreadyRanToday = DateUtils.isSameDay(lastRun, today);
+        if ( alreadyRanToday ) return false;
```

Comment:
Let's cleanup futher:
```suggestion
        if ( DateUtils.isSameDay(lastRun, today) ) return false;
```